src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -177,19 +180,19 @@ class PeerActor(
       daoBlockHeaderOpt match {
         case Some(_) if daoForkValidator.validate(msg).isEmpty =>
           log.info("Peer is running the ETC chain")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
 
         case Some(_) if nodeStatusHolder().blockchainStatus.totalDifficulty < daoForkBlockTotalDifficulty =>
           log.info("Peer is not running the ETC fork, but we're not there yet. Keeping the connection until then.")
-          context become new HandshakedHandler(rlpxConnection, remoteStatus).receive
+          context become new HandshakedHandler(rlpxConnection, remoteStatus, daoForkBlockNumber).receive
```

Comment:
nice catch @adamsmo I think the only protection we currently have is to have at least X peers before starting the sync (and hope at least one of them is etc chain after the fork). We definately need a ticket to investigate/fix (if needed) that.